Modules/Core/ImageFunction/include/itkRayCastInterpolateImageFunction.hxx

```diff
@@ -1416,10 +1416,17 @@ RayCastInterpolateImageFunction<TInputImage, TCoordRep>::Evaluate(const PointTyp
   ray.ZeroState();
   ray.Initialise();
 
-  PointType origin = this->m_Image->GetOrigin();
-  for (unsigned int i = 0; i < origin.Length; ++i)
-    origin[i] -= 0.5 * this->m_Image->GetSpacing()[i];
-  ray.SetRay(point - origin, direction);
+  const PointType origin = this->m_Image->GetOrigin();
+  const auto      spacing = this->m_Image->GetSpacing();
```

Comment:
The type isn't apparent from the right-hand side, so I'd put `SpacingType` here instead of `auto`.

Reply:
@Leengit Not entirely convinced. For a variable that is already named `spacing`, initialized by `GetSpacing()`, to explicitly specify that its type is `SpacingType` doesn't seem very informative to me.  🤔